python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -309,6 +309,7 @@ def _scheduling_loop_step(self, topology: Topology) -> bool:
             i += 1
             if i % PROGRESS_BAR_UPDATE_INTERVAL == 0:
                 self._refresh_progress_bars(topology)
+            op.update_resource_usage()
```

Comment:
let's move this call inside `ResourceManager.update_usages()` [here](https://github.com/ray-project/ray/blob/7c220a5eadda4f0690ef5642ad750ae5929602df/python/ray/data/_internal/execution/resource_manager.py#L130)
Because the updated info will be used in that function